strongbox-parent/pom.xml

```diff
@@ -396,6 +396,12 @@
                 <groupId>com.orientechnologies</groupId>
                 <artifactId>orientdb-object</artifactId>
                 <version>${version.orientdb}</version>
+                <exclusions>
```

Comment:
there is `hibernate` dependency there?

Reply:
Yes. See details here: http://mavenrepository.com/artifact/com.orientechnologies/orientdb-object/2.2.18

Reply:
confirmed :) thanks !